scenario/runtime.py

```diff
@@ -298,6 +480,8 @@ def exec(
         This will set the environment up and call ops.main.main().
         After that it's up to ops.
         """
+        ConsistencyChecker(state, event, self._charm_spec, self._juju_version).run()
```

Comment:
Shouldn't a class that gets created to call a single method and then the instance is thrown away just be a stand-alone function?

Reply:
yeah, good point. Will refactor that.